.agents/skills/cloudflare-email-service/references/routing.md

```
@@ -0,0 +1,199 @@
+  console.log("Text:", parsed.text);
+  console.log("Attachments:", parsed.attachments.length);
+}
+```
+
+## Store and Reply Later (Human-in-the-Loop)
+
+A common pattern is to store incoming emails in a Durable Object (SQLite) so a human or AI agent can review and reply later — rather than replying immediately in the `email()` handler. This enables support inboxes, approval workflows, and AI-drafted replies.
+
+### Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)